auth/server/src/main/java/com/seagates3/authentication/ClientRequestParser.java

```diff
@@ -94,7 +94,8 @@ public static ClientRequestToken parse(FullHttpRequest httpRequest,
 
         if (authorizationHeader == null ||
             authorizationHeader.replaceAll("\\s+", "").isEmpty()) {
-            return null;
+          ServerResponse serverResponse = responseGenerator.AccessDenied();
+          throw new InvalidArgumentException(serverResponse);
```

Comment:
Does caller handle this exception

Reply:
yes. IAMController is handling this exception